projects/orquestra-sdk/docs/examples/tests/test_dependency_installation.py

```diff
@@ -197,7 +197,9 @@ def test_good_practice_defaults():
         # Then
         assert src_import.type == "INLINE_IMPORT"
         assert len(deps_imports) == 1
-        assert deps_imports[0].type == "PYTHON_IMPORT"
+        # it depends if you have installed SDK with version from PyPi, or
+        # its local-dev version, it changes what import is used for SDK
+        assert deps_imports[0].type == "PYTHON_IMPORT" or "GIT_IMPORT"
```

Comment:
**issue**: that's not how equality works in Python 😄 

```python
>>> foo = "foo"
>>> foo == "abc" or "def"
'def'
```

```suggestion
        assert deps_imports[0].type in {"PYTHON_IMPORT", "GIT_IMPORT"}
```

Reply:
🤦